pkg/pci/devices.go

```diff
@@ -38,3 +38,26 @@ func (d Devices) ReadConfig() error {
 	}
 	return nil
 }
+
+// ReadConfigRegister reads the config info for all the devices.
```

Comment:
what are r and s? call them offset and size?

Reply:
reg, size. Done.